solution/ui/regulations/eregs-vite/src/components/resources/ResourcesResults.vue

```diff
@@ -7,33 +7,19 @@
                     <div class="result-label category-label">
                         {{
                             item.category.parent
```

Comment:
You made changes to this file, but it looks like you're not using it in any of your new components.